### ISSUE TITLE
Rows automatically adjusting height

### ISSUE THREAD

**Issue by PrymeWolf** [awSweeney/FFXIV-ACT-Rainbow-Mage-Overlays#6]
On the overlay, depending on the jobs the height of the rows are different. I've noticed this on WHM, SGE, RPR, SCH and some others to be different height.

**Comment by awSweeney** [comment]
Hi @PrymeWolf , can you perhaps include a screenshot of what you're seeing? I'm not seeing this behavior.

**Comment by awSweeney** [comment]
Issue confirmed. 

Appears to be a result of the SCH icon being an incorrect size. NIN and SMN icons found to be a couple pixels off as well. Adjusting icons to 32x32.

**Comment by PrymeWolf** [comment]
upon changing the pixels to 32x32, the SCH icons specifically looks very distorted and enlarged.
![image](https://user-images.githubusercontent.com/95669152/145044922-69faddc3-563d-4245-be32-ac4e102da304.png)

**Comment by awSweeney** [comment]
Sch icon adjusted further.

![dfaabad1b8](https://user-images.githubusercontent.com/5041783/145302243-abf57945-4b9c-4b1f-b35e-65c8cbd22832.png)

**Comment by PrymeWolf** [comment]
Maybe it's just me, but do you know why my rows are still different heights?
![image](https://user-images.githubusercontent.com/95669152/145445556-a104a2ce-2788-4ea1-85fa-84681c06e1f2.png)

**Comment by awSweeney** [comment]
In this scenario it may be due to the broken image link. I noticed this has started popping up since Endwalker when it cannot find the correct entity. I opened a separate issue and currently testing out a solution.

Do you recall what class Mituking was on during this encounter? I have not seen the broken link occur for any of the player classes. Usually it has been limited to NPC entities.

**Comment by PrymeWolf** [comment]
I believe Mituking was playing RDM.

**Comment by Thalassocrat** [comment]
Also getting thicker rows than usual or than the sample pictures on this page. Just thought I'd add an example. Thanks for the overlay!
https://puu.sh/IveEr/743614cb69.png

**Comment by awSweeney** [comment]
Thanks Thalassocrat. That helps confirm it's not an isolated issue. Will get to the bottom of it, unfortunately the long queues are causing delays in dev/testing. Thanks for your patience :)

**Comment by awSweeney** [comment]
@Thalassocrat and @PrymeWolf I pushed an update to the Endwalker branch, setting the bars to a fixed height. Can you see if it makes a difference for you?

**Comment by Thalassocrat** [comment]
I'm sorry it's still the same for me.

**Comment by awSweeney** [comment]
It would seem the styles are being ignored.

Just to confirm:

- Try doing a fresh install, deleting the previous version and starting with a fresh directory.
- The new downloaded zip file reads FFXIV-ACT-Rainbow-Mage-Overlays-**Endwalker-v1.1.1**.zip or FFXIV-ACT-Rainbow-Mage-Overlays-**Endwalker-Updates**.zip
- The theme selected in ACT is {Install-Dir}/FFXIV-ACT-Rainbow-Mage-Overlays/overlays/MiniParseGlow_CritMonitor_Classic.html
- The DLL files included in the ACT Overlay Plugin are unblocked as outlined in ReadMe step number 2
- Running ACT as admin and on the latest version
- What OS are you running on Windows 10/11, Mac?

**Comment by awSweeney** [comment]
@dconns1 mind poking around and see if you can recreate on your side?

**Comment by PrymeWolf** [comment]
It's working correctly now, thanks.

I see that due to this change I am unable to customise the overlay. Is there a way I can customise it?
Thanks

**Comment by Thalassocrat** [comment]
It is the most fixed, thank you SO much for your overlay and time. It is great, and very appreciated.

**Comment by awSweeney** [comment]
Excellent. Thank you for your patience and communication. I will mark this as resolved. If anything else pops up please feel free to open a new issue.

Prymewolf I saw a brief comment asking about customization. The styles were moved under the CSS folder. You should be able to customize as before by modifying there. I had moved things around a bit to be able to add some new overlays in the future.

**Comment by awSweeney** [comment]
Resolved under v1.1.1
